test/extras/no_use_pycon.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2019 Snowflake Computing Inc. All right reserved.
```

Comment:
nit; 2020

Reply:
Done, sorry I named the commit wrongly here but it's going to be squashed anyway